src/main/java/uk/gov/hmcts/juror/job/execution/jobs/contentstore/ContentStoreFileJob.java

```diff
@@ -171,4 +169,14 @@ public ResultSupplier getResultSupplier() {
             metaData -> uploadFiles()
         ));
     }
+
+    private void updateDateSent(File file) {
+        databaseService.execute(getDatabaseConfig(), connection -> {
+            try {
+                databaseService.executeUpdate(connection, UPDATE_SQL_QUERY, file.getName(), fileType);
+            } catch (SQLException e) {
+                log.error(fileType + ": Failed to update file: " + file.getName(), e);
```

Comment:
Need to report this result back so the schedular has the metrics for it.
Will need to add the failed to update name to meta data and fail the result.

Reply:
Done